Makefile

```diff
@@ -27,7 +27,7 @@ dev: install
 test: testacc
 
 testacc:
-	test/run_tests.sh
+	test/example-tests.sh
```

Comment:
Can you change this back?  This is not corect.

Reply:
done